src/components/Paint.tsx

```diff
@@ -482,7 +482,9 @@ function Paint(props: PaintProps) {
                         touchPos.current = lastTouchPos;
                     }}
                     onWheel={e => {
-                        // TODO: Use e.deltaY to zoom into the canvas?
+                        const newScale = clamp(scale + e.deltaY * 0.001, 0.5, 2);
```

Comment:
how did we get these numbers?

also just putting this [link](https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event) in for onWheel documentation for my reference.

also.. why is it `scale + e.delta.Y` I feel like the any scaling should be multiplied into the new scale?

why is .5 the min? why is 2 the max? why are we multiplying the e.deltaY by 0.001? is it because the d

Reply:
These are all just temporary numbers while I work out the full functionality of it all. Once I have everything figured out I'll turn them into numbers that take into account the canvas' dimensions and stuff